packages/slider/README.md

```diff
@@ -206,84 +355,41 @@ Any number (including `0`) can be used as a fill-start value. If a [custom norma
 ></sp-slider>
 ```
 
-### Ramp
-
-```html
-<sp-slider label="Slider Label" variant="ramp"></sp-slider>
-<sp-slider label="Slider Label - Disabled" variant="ramp" disabled></sp-slider>
-```
-
-### Editable
-
-An `<sp-slider>` element can be paired with an `<sp-number-field>` element via the `editable` attribute. The `<sp-number-field>` will be passed all of the standard options from the `<sp-slider>` element (e.g. `min`, `max`, `formatOptions`, etc.) and will also accept the `hide-stepper` attribute in order to prevent the display of its stepper UI.
-
-```html
-<sp-slider
-    label="Angle (editable)"
-    editable
-    hide-stepper
-    min="0"
-    max="360"
-    format-options='{
-        "style": "unit",
-        "unit": "degree",
-        "unitDisplay": "narrow"
-    }'
-></sp-slider>
-<sp-slider
-    label="Hours of the day (editable)"
-    editable
-    max="24"
-    min="0"
-    value="7.25"
-    step="0.25"
-    style="--spectrum-slider-editable-number-field-width: 100px;"
-    format-options='{
-        "style": "unit",
-        "unit": "hour"
-    }'
-></sp-slider>
-```
-
-#### Quiet
-
-```html
-<sp-slider quiet editable value="50"></sp-slider>
-<sp-slider quiet disabled editable value="50"></sp-slider>
-```
+</sp-tab-panel>
+</sp-tabs>
 
-#### Default value
+#### Ramp
 
-Slider will reset to its `default-value` when the user double clicks on the slider handle or if the user presses the `escape` key when the slider handle is focused.
+With `variant="ramp"`, the slider with increases as it approaches the `max` position.
 
 ```html
-<sp-slider value="50" default-value="20"></sp-slider>
+<sp-slider label="Slider Label" variant="ramp"></sp-slider>
+<sp-slider label="Slider Label - Disabled" variant="ramp" disabled></sp-slider>
 ```
 
-Note: If a slider with `default-value` attribute is contained in a modal and the slider-handle is focused then the following interaction will occur on pressing the `escape` key:
-
--   If the slider value is different from the default value then the slider value will be reset to the default value and the modal will not be closed.
--   If the slider value is equal to the default value then the modal will be closed.
-
-#### Indeterminate
+### Range
```

Comment:
Moved this example from slider handle as `range` is technically a slider `variant`.